sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 ### Added
 - Added CommunicationIdentityClient (originally was part of the azure.communication.administration package).
+- Added ability to create a user and issue token for it at the same time.
+
+### Breaking
+- CommunicationIdentityClient.RevokeTokens tokens now revoke all the currently issued tokens instead of revoking tokens issued prior to a given time.
+- CommunicationIdentityClient.IssueToken returns an instance of `Azure.Core.AccessToken` instead of `CommunicationUserToken`.
```

Comment:
not sure if this is pythonic for docs, should it be azure.core.credentials.AccessToken instead of Azure.Core.AccessToken?

Reply:
Yeah, I'll update this.